draft-ietf-ccamp-yang-otn-slicing.md

```diff
@@ -466,6 +467,65 @@ shall support configuring an OTN slice with both options.
    - together with a higher-level network slice controller to support 
       end-to-end network slicing;
 
+# Realizing OTN Slices
+
+[I-D.ietf-teas-ietf-network-slices] introduces a mechanism for an IETF network slice controller to realize network slices by constructing Network Resource Partitions (NRP). A NRP is a collection of resources identified in the underlay network to facilitate the mapping of network slices onto available network resources. An NRP is a scope view of a topology and may be considered as a topology in its own right. Thus, in traffic-engineered (TE) networks including OTN, an NRP may be simply represented as an abstract TE topology defined by [RFC8795]. For OTN networks, An NRP may be represented as an abstract OTN topology defined by [I-D.ietf-ccamp-otn-topo-yang].
+
+The NRP may be used to address the scalability issues where there may be considerable numbers of control and data plane states required to be stored and programmed if network slices are mapped directly to the underlay topology. NRP is internal to a network slice controller, and use of NRPs is optional yet could benefit a network slice realization in large-scale networks, including OTN networks. 
+
+For connectivity-based OTN slices, a connection within an OTN slice can be realized by an OTN tunnel in the underlay topology and resources are reserved by the tunnel, thus use of NRP is optional in this case. 
+
+For resource-based OTN slices, the OTN-SC may map an OTN slice directly onto the underlay TE topology presented by the subtended network controller (MDSC or PNC) without creating NRP topologies. Due to the need for reserving resources, the OTN-SC needs to color corresponding link resources of the underlay topology with a slice identifier and maintain the coloring to keep track of the mapping of OTN slices. The OTN-SC may push the colored topology to the subtended MDSC or PNC using the MPI model defined in this draft.
+
+Alternatively, an OTN slice may be mapped to a NRP as an overlay abstract OTN TE topology on top of the underlay topology. The corresponding link resources allocated to the slice is encapsulated in and tracked by the abstract topology, and a given link or port in the NRP topology represents resources that are reserved in the underlay topology. One OTN slice is typically realized by one dedicated NRP topology and all the resources within that NRP topology are reserved for the OTN slice. In this case, the use of NRP eliminates the need for coloring links in the underlay topology, and the NRP topology may be pushed directly to the subtended MDSC or PNC by the OTN-SC.
```

Comment:
In the YANG model, it is possible to define request more than one topology for one resource-based OTN slice. In this case, I think we will have one NRP for each slice topology rather than for each resource-based OTN slice

OLD

> One OTN slice is typically realized by one dedicated NRP topology


NEW

> One slice topology for a resource-based OTN slice is typically realized by one dedicated NRP topology